style/app.scss

```diff
@@ -2250,3 +2250,43 @@ main-filter input {
 #nav-generate {
   white-space: nowrap;
 }
+
+.error-overlay {
```

Comment:
Having some custom CSS is fine, but look to match it with the existing CSS for modals (the course info modal, the export modal)